client/src/appsupport.js

```diff
@@ -137,9 +137,9 @@ function displayError(msg) {
 }
 
 window.onerror = function (msg, url, lineno, colno, error) {
+  console.error("Uncaught exception", msg, url, lineno, colno, error);
   window.Rollbar.error(msg, error);
   window.lastError = error;
-  console.error("Uncaught exception", msg, url, lineno, colno, error);
```

Comment:
Is this dumb? My theory is that Rollbar.error might leave some parts of `erro` out - maybe the console.error in the logs may be visible in the "telemetry" section of rollbar, making some things easier to diagnose.

Reply:
I think you gotta feel it out for a bit before knowing for sure.